grails-app/domain/org/pih/warehouse/product/ProductSupplier.groovy

```diff
@@ -127,6 +130,46 @@ class ProductSupplier implements Serializable, Comparable<ProductSupplier> {
                         id <=> obj.id
     }
 
+    String getPackageSize() {
+        // First we check for a default package assigned to 1:1 association
+        if (defaultProductPackage) {
+            return "${defaultProductPackage?.uom?.code}/${defaultProductPackage?.quantity}"
+        }
+        // If a default package of 1:1 association is not found, check for a default package "in an old way" (last updated)
+        if (defaultProductPackageDerived) {
+            return "${defaultProductPackageDerived?.uom?.code}/${defaultProductPackageDerived?.quantity}"
+        }
+        return null
+    }
+
+    BigDecimal getPackagePrice() {
+        // First we check for a default package assigned to 1:1 association
+        if (defaultProductPackage) {
+            return defaultProductPackage?.productPrice?.price?.setScale(2, RoundingMode.HALF_UP) ?: 0.0
+        }
+        // If a default package of 1:1 association is not found, check for a default package "in an old way" (last updated)
+        if (defaultProductPackageDerived) {
+            return defaultProductPackageDerived?.productPrice?.price?.setScale(2, RoundingMode.HALF_UP) ?: 0.0
+        }
+        return 0.0
+    }
+
+    BigDecimal getUnitPrice() {
+        // First we check for a default package assigned to 1:1 association
+        if (defaultProductPackage) {
+            return defaultProductPackage?.productPrice
+                    ? (defaultProductPackage?.productPrice?.price / defaultProductPackage?.quantity)?.setScale(2, RoundingMode.HALF_UP)
+                    : 0.0
+        }
+        // If a default package of 1:1 association is not found, check for a default package "in an old way" (last updated)
+        if (defaultProductPackageDerived) {
+            return defaultProductPackageDerived?.productPrice
+                    ? (defaultProductPackageDerived?.productPrice?.price / defaultProductPackageDerived?.quantity)?.setScale(2, RoundingMode.HALF_UP)
+                    : 0.0
+        }
+        return 0.0
+    }
+
```

Comment:
lines 133-172 (and this commit in particular https://github.com/openboxes/openboxes/pull/4522/commits/fd98c02435f2c49683cfac23b45ce23a4a97d7dc) are applying the new behavior of the default product package to the product sources list.
At first we wanna search for a default product package that is set in the new association `ProductPackage defaultProductPackage` and if we do not find one, then we wanna search for the default product package "in an old way" to support backwards compatibility. 
It has been specified and requested in the comments of OBPIH-6116.